cluster/src/main/java/com/hartwig/pipeline/tertiary/linx/Linx.java

```diff
@@ -101,6 +101,7 @@ public LinxOutput output(final SomaticRunMetadata metadata, final PipelineStatus
                 .status(jobStatus)
                 .maybeLinxOutputLocations(LinxOutputLocations.builder()
                         .breakends(GoogleStorageLocation.of(bucket.name(), resultsDirectory.path(breakendTsv)))
+                        .drivers(GoogleStorageLocation.of(bucket.name(), resultsDirectory.path(driversTsv)))
```

Comment:
I can't comment on it directly because it's not part of this review, but in April I added line 110/111, which doesn't look right to me anymore. Maybe we need to make another change there, I can't remember why that was done.